packages/design-system/src/components/UsaBanner/UsaBanner.tsx

```diff
@@ -25,43 +33,104 @@ export interface UsaBannerProps {
 
 export const UsaBanner: React.FunctionComponent<UsaBannerProps> = (props: UsaBannerProps) => {
   const [isBannerOpen, setBannerOpen] = useState<boolean>(false);
-
-  const toggleBanner = () => {
-    setBannerOpen(!isBannerOpen);
-  };
-
+  const [shouldRenderMobileView, setShouldRenderMobileView] = useState<boolean>(false);
   const t = props.locale === 'es' ? SpanishTranslations.usaBanner : EnglishTranslations.usaBanner;
   const langProp = props.locale === 'es' ? 'es' : null;
   const classes = classNames('ds-c-usa-banner', props.className);
   const id = props.id || uniqueId('gov-banner_');
+  let media;
 
-  return (
-    <section className={classes} aria-label={t.bannerLabel} lang={langProp}>
-      <header
-        className={`ds-c-usa-banner__header ${
-          isBannerOpen ? 'ds-c-usa-banner__header--expanded' : ''
-        }`}
+  const onMediaChange = (evt) => {
+    setShouldRenderMobileView(evt.matches);
+  };
+
+  useEffect(() => {
+    if (window) {
+      const media = window.matchMedia('(max-width: 543px)');
+      media.addEventListener('change', onMediaChange);
+
+      setShouldRenderMobileView(media.matches);
+    }
```

Comment:
Defining this inside the `useEffect` here might make the relationship clearer. It doesn't look like it's used anywhere else in the component